src/ui/public/vislib/lib/axis/axis.js

```diff
@@ -266,7 +163,7 @@ export default function AxisFactory(Private) {
 
     validate() {
       if (this.axisConfig.isLogScale() && this.axisConfig.isPercentage()) {
-        throw new errors.VislibError(`Can't mix percentage mode with log scale.`);
```

Comment:
Not incredibly pertinent, but using the back-tick over single-quotes is recommended here.